awscli/examples/translate/list-languages.rst

```diff
@@ -0,0 +1,62 @@
+**To list supported languages**
+
+The following ``list-languages`` example lists all languages supported by Amazon Translate. ::
+
+    aws translate list-languages
+
+Output::
+
+    {
+        "Languages": [
+            {
+                "LanguageName": "Afrikaans",
+                "LanguageCode": "af"
+            },
+            {
+                "LanguageName": "Albanian",
+                "LanguageCode": "sq"
+            },
+            {
+                "LanguageName": "Amharic",
+                "LanguageCode": "am"
+            },
+            {
+                "LanguageName": "Arabic",
+                "LanguageCode": "ar"
+            },
+            {
+                "LanguageName": "Armenian",
+                "LanguageCode": "hy"
+            },
+            {
+                "LanguageName": "Azerbaijani",
+                "LanguageCode": "az"
+            }
+        ]
+    }
+
+**To list languages with a specific display language**
+
+The following ``list-languages`` example lists supported languages with names displayed in German. ::
+
+    aws translate list-languages \
+        --display-language-code de
+
+Output::
+
+    {
+        "Languages": [
+            {
+                "LanguageName": "Afrikaans",
+                "LanguageCode": "af"
+            },
+            {
+                "LanguageName": "Albanisch",
+                "LanguageCode": "sq"
+            },
+            {
+                "LanguageName": "Amharisch",
+                "LanguageCode": "am"
+            }
+        ]
+    }
```

Comment:
Please provide a "For more information" link for every example in this request. See format in the contribution guide: [aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link](https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link)